code/modules/jobs/job_types/captain.dm

```diff
@@ -106,7 +107,16 @@
 
 /datum/outfit/job/captain/post_equip(mob/living/carbon/human/equipped, visualsOnly)
 	. = ..()
-	if(visualsOnly || !special_charter)
+	if(visualsOnly)
+		return
+
+	var/obj/item/door_remote/omni/cap_remote = locate() in equipped.get_item_by_slot(ITEM_SLOT_BACK)
+	if(isnull(cap_remote))
+		// failed to give out the remote, plop on the ground
+		cap_remote = new(get_turf(equipped))
+	cap_remote.name = "Captain's door remote"
```

Comment:
Why is this necessary?

Reply:
The balance change or this bit where it gets put in the backpack?

Reply:
This code to put it in the backpack and rename it. You already use the `backpack_contents` list, so why is this here?